src/main/java/com/ncedu/cheetahtest/service/security/AuthServiceImpl.java

```diff
@@ -67,7 +67,7 @@ public AccessTokenDto login(LoginDto loginDto) {
 
         String loginDtoPasswordWithSalt = loginDto.getPassword() + passwordSalt;
 
-        if (!passwordEncoder.matches(loginDtoPasswordWithSalt, user.getPass())) {
+        if (!passwordEncoder.matches(loginDtoPasswordWithSalt, user.getPass()) || !user.getStatus().equals("active")) {
```

Comment:
Please. move user statuses to enum.
Use Yoda conditions when compare something with a constant value